### DISCUSSION TITLE
ME168 over 150°C on auto

### DISCUSSION THREAD

**Discussion by SirUlbrich** [Koenkk/zigbee2mqtt#19757] [Devices]
what did i'm doing wrong?
on auto i guess its going to do schedule.

but my setpoint take up to over 150°C
![failure](https://github.com/Koenkk/zigbee2mqtt/assets/134588380/971e57dd-2514-4fb5-8d84-abced4b997f0)

**Comment by void1042** [comment]
Related issue: https://github.com/Koenkk/zigbee2mqtt/issues/19787
